enforcer-rules/README.md

```diff
@@ -0,0 +1,89 @@
+# Linkage Checker Enforcer Rule
+
+This Maven enforcer rule verifies that the transitive dependency tree of `pom.xml` does not have
+any [linkage error](../library-best-practices/glossary.md#types-of-conflicts-and-compatibility).
+
+## Class path and dependencySection flag
+
+The value of the dependencySection element determines whether the rule checks the dependencies in
+the `dependencies` section or the `dependencyManagement` section.
+
+- When `DEPENDENCIES` (default value), the rule checks the class path calculated from the project's
```

Comment:
if here you change this to

"When the value of the dependencySection element is DEPENDEnCIES"

Reply:
Updated to more simpler form. Does this look good?